LICENSE.md

```diff
@@ -24,86 +24,17 @@ The Convex.jl package is licensed under the Simplified "2-clause" BSD License:
 > (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 > OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-The file src/problem_depot/problem_depot.jl contains some utilities copied from
-the MathOptInterface.jl package (https://github.com/jump-dev/MathOptInterface.jl)
-which is licensed under the following MIT License:
-
->Copyright (c) 2017: Miles Lubin and contributors Copyright (c) 2017: Google Inc.
->
->Permission is hereby granted, free of charge, to any person obtaining a copy
->of this software and associated documentation files (the "Software"), to deal
->in the Software without restriction, including without limitation the rights
->to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
->copies of the Software, and to permit persons to whom the Software is
->furnished to do so, subject to the following conditions:
->
->The above copyright notice and this permission notice shall be included in all
->copies or substantial portions of the Software.
->
->THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
->IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
->FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
->AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
->LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
->OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
->SOFTWARE.
-
-The files benchmark/pprinthelper.jl, benchmark/pprintjudge.jl,
-benchmark/pprintresult.jl, benchmark/runbenchmarks.jl, and benchmark/runjudge.jl
-and .travis.yml contain code copied from the Transducers.jl package
-(https://github.com/tkf/Transducers.jl/) which is licensed under the MIT License:
-
->Copyright (c) 2018 Takafumi Arakaki
```

Comment:
I've moved this to a separate LICENSE.md file in `benchmark`. It isn't part of the Convex.jl source code that gets loaded by `using Convex`.